leaf-ui/js/next-state.js

```diff
@@ -197,6 +197,14 @@
         var pagination = document.getElementById("pagination");
         var num_states_lbl = document.getElementById("num_states_lbl");
         var currentPageIn = document.getElementById("currentPage");
+
+        // disable Explore Next States button if last time point
+        var exploreNextStatesBtn = document.getElementById("exploreNextStates");
```

Comment:
If the last time point is selected, the Explore Next States button is disabled. However, it is not re-enabled when a different, earlier time point is selected. This remains even when you load a new Explore States window from the same time point.
<img width="202" alt="Screen Shot 2023-10-19 at 1 08 23 PM" src="https://github.com/amgrubb/BloomingLeaf/assets/59470488/4b7b8a68-8d51-488a-b212-97afe4bbc7ca">

Reply:
@YesugenB We are having trouble recreating this issue. Could you clarify what you mean by "selecting an earlier timepoint." Do you mean closing the next state tab and reopening a new one at an earlier timepoint from the analysis tab? Or is there a way to get to an earlier timepoint from the next state window?